Gemfile

```diff
@@ -80,7 +77,7 @@ group(:omnibus_package) do
 
   # test-kitchen and plugins
   gem "test-kitchen", ">= 2.0"
-  gem "kitchen-azurerm", ">= 0.14"
+  gem "kitchen-azurerm", ">= 0.14", "< 0.16.0"
```

Comment:
Any reason not to bring in the improvements in 0.16?

Reply:
Nope, removing this constraint